.github/workflows/kind.yaml

```diff
@@ -0,0 +1,16 @@
+name: KinD Conformance Tests 
+on:
+  pull_request: {}
+  push: {}
+jobs:
+  kind_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: engineerd/setup-kind@v0.4.0
+    - name: Preparing config
+      run: |
+          mv .github/config.yaml .
+    - name: Running conformance tests in KinD cluster
+      run: |
+          go test -timeout 1h -ginkgo.v
```

Comment:
Wondering how this would work, for the ingress test-case as there is no externalIP being created? :thinking:

Reply:
As discussed over Slack, we have a custom `config.yaml` for KinD, so that we can skip ingress on kind - https://github.com/linkerd/linkerd2-conformance/blob/95c1e3b0c192e5f655cc6f50436841a7c4776004/.github/config.yaml#L22